tools/meiliExporter.go

```diff
@@ -106,6 +123,32 @@ func (m *MeiliExporter) Export() {
 			logger.Error("issue adding documents to meili", "err", err)
 		}
 	})
+
+	coursesIndex := m.c.Index("COURSES")
+	_, err = coursesIndex.DeleteAllDocuments()
+	if err != nil {
+		logger.Warn("could not delete all old courses", "err", err)
+	}
+
+	m.d.CoursesDao.ExecAllCourses(func(courses []dao.Course) {
+		meilicourses := make([]MeiliCourse, len(courses))
+		courseIDs := make([]uint, len(courses))
+		for i, course := range courses {
+			courseIDs[i] = course.ID
```

Comment:
This seems to be dead code or at least debugging code

```suggestion
		for i, course := range courses {
```

Reply:
Yes, this is a relic of me copying old code without thinking about it too much. I won't commit your suggestion and instead delete this here too ;)
https://github.com/TUM-Dev/gocast/blob/af18af51725468c0b118c252e4927990ac0163c4/tools/meiliExporter.go#L76-L78